File: python/examples/example_vessels.py
from vesselfinder_api import VesselFinderApi
from vesselfinder_api.exceptions import ApiErrorException

v = VesselFinderApi(userkey='-- Input your userkey here --', errormode=False, save_last_info=True)

try:
    print(v.vessels(imo=[9228801,9441271], mmsi=227441980))
except ApiErrorException as e:
    print(e)

File: php/examples/example_vessels.php
<?php
include __DIR__ . '/../VesselFinderApi.php';

use API\VesselFinderApi;

$client = new VesselFinderApi('-- Input your userkey here --');

try {
    $vessels = $client->vessels([9228801,9441271], 227441980);
    print_r($vessels);
} catch (\Exceptions\ApiErrorException $e) {
    print_r($e->getMessage() . PHP_EOL);
}